### ISSUE TITLE
[World] North American Rivers

### ISSUE THREAD

**Issue by darthgrape2** [EarthMC/Issue-Tracker#48]
- [ ] Mississippi
- [ ] Missouri
- [ ] Ohio
- [ ] Illinois
- [ ] Tennessee
- [x] Red
- [x] Arkansas
- [x] Acha

The rivers in North America exist, but are too small for boat travel.  At most they are 3-4 blocks wide.  At some sections, a river can completely disappear and reappear around 10 blocks later.  This makes it hard for players to travel, since direct teleportation is disabled and only nation homes' can be teleported too.
![image](https://user-images.githubusercontent.com/44533906/47609058-0c8c9980-da06-11e8-8ed0-dda4222b0250.png)
These are the major rivers that in my opinion, should be widened for easier travel
![louisianna](https://user-images.githubusercontent.com/44533906/47609080-57a6ac80-da06-11e8-976e-afa328178469.JPG)
This is the mouth of the Mississippi River on TerraNova.  As it comes from the North, it slowly gets skinnier and disappears, before getting into the Gulf Of Mexico.
![mississippi](https://user-images.githubusercontent.com/44533906/47609135-e4ea0100-da06-11e8-8f8b-77e00bdc4332.JPG)
This is what the river should look like when it reaches the coast.  This is just one example of all the rivers in North America.  What I have heard from donators who have made towns in North America is that the Hudson River and Delaware River also are not navigable. 
![emcriver](https://user-images.githubusercontent.com/44533906/47609294-d5b88280-da09-11e8-82f2-40321076b74b.JPG)
This is what the rivers looked like on EMC, I think this size would be a good fit for major rivers, like the Mississippi, Rio Grande, and Hudson.  Rivers that branch off of these should be smaller, but only by a couple of blocks.

Basically, widen all the rivers to around 10 blocks so that they can be used for travel and trade

**Comment by Arganog** [comment]
Addition screenshot from previous map, I also advocate that these rivers would be great to have updated. (I was about to make an issue to include the Red River, but it is included in these screenshots)

![20181027_155750](https://user-images.githubusercontent.com/44342952/47609417-b0287a80-da03-11e8-90b0-10cc86c25261.jpg)

**Comment by shirazmatas** [comment]
Yes will do when finished with poland

**Comment by sharkfynn1012** [comment]
Yes! The rivers in North America need some work. The Colorado river too.

**Comment by Arganog** [comment]
On the mississippi river, the town of St Louis (around -16700 -7000) will likely be right on top of river expansion efforts

The owner, DrBubonic, is aware and asks that we just refund anything we need to destroy.

![stlouis](https://user-images.githubusercontent.com/44342952/48075252-69c3e000-e1a8-11e8-9209-4a99f05d5194.PNG)
